keyboards/mschwingen/modelm/rules.mk

```
@@ -0,0 +1,43 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = lufa-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+CUSTOM_MATRIX = lite
+KEY_LOCK_ENABLE = yes
+
+DYNAMIC_MACRO_ENABLE = yes
+
+UART_DEBUG = no
+
+SRC += matrix.c
+SRC +=	$(COMMON_DIR)/uart.c
+QUANTUM_LIB_SRC += spi_master.c
```

This should also probably be quantum, since it's timing related. 

```suggestion
QUANTUM_LIB_SRC += $(COMMON_DIR)/uart.c \
                   spi_master.c
```